101-ubuntu-mate-desktop-vscode/azuredeploy.json

```diff
@@ -262,8 +245,8 @@
         "autoUpgradeMinorVersion": true,
         "settings": {
           "fileUris": [
-            "[concat(parameters('_artifactsLocation'), '/', variables('ScriptFileName'), parameters('_artifactsLocationSasToken'))]"
-          ]
+            "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-ubuntu-mate-desktop-vscode/infra.sh"
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts

Reply:
@bmoore-msft  thanks changes made and new PR